app/controllers/search_controller.rb

```diff
@@ -25,6 +27,10 @@ def show
       @demands = user_municipality.state.demands.to_a
     end
 
+    if params[:topic].present? && Constants::DEMAND_TOPICS.include?(params[:topic])
```

Comment:
This is going to do an in-memory sort. Would be great to get this filtering set up before issuing the SQL query. Maybe I can tackle that in #20